src/index.ts

```diff
@@ -18,8 +18,18 @@ import { menu } from './menu';
 import { ImageExtensionService } from './image-extension-service';
 import { ImageGenerationService } from './image-generation-service';
 import { ImageUploadService } from './image-upload-service';
+import { FRONTEND_HELPER } from './frontend-helper';
 
 menu;
 ImageExtensionService;
 ImageGenerationService;
 ImageUploadService;
+FRONTEND_HELPER;
+
+function doGet() {
```

Comment:
I would consider moving these two function to a separate file, like we did with the `menu` above.

Reply:
Done.